dingo/backup/redundant_facet_removal_backup.py

```diff
@@ -0,0 +1,643 @@
+# dingo : a python library for metabolic networks sampling and analysis
+# dingo is part of GeomScale project
+
+# Copyright (c) 2021 Apostolos Chalkis
+
+# Licensed under GNU LGPL.3, see LICENCE file
+
+import sys
+import numpy as np
+import scipy.sparse as sp
+import gurobipy as gp
+from gurobipy import GRB
+import math
+
+
+def fast_fba(lb, ub, S, c):
```

Comment:
Could you plz remove fast_fba() and fast_fva() from this file since they are irrelevant?